### ISSUE TITLE
Show reason for transition

### ISSUE THREAD

**Issue by melborp** [melborp/StateModelVisualization#13]

**Comment by melborp** [comment]
No API still, so postponing the feature

**Comment by sitr** [comment]
Anything new here? I would really like to see the reason for transitions. That was the whole idea when I got this extension installed in our Azure Devops.

**Comment by melborp** [comment]
Yes, the api offers actions now. We can use WorkItemStateTransition.actions. 
Need to experiment if showing these on edges or as tooltips is better. 
Should be toggleable on and off.

**Comment by melborp** [comment]
Seems i was too optimistic. Need some investigation with api to see what is shown.
In process editor - you cant really define a reason with states.
